docs/pages/docs.html

```diff
@@ -102,10 +102,10 @@ <h2 class="accordion-header" id="flush-heading{{docs.number}}">
                     <h4 class="doc-heading">{{ docs.name }}</h4>
                   </button>
                 </h2>
-                  <div id="flush-collapse{{docs.number}}" class="accordion-collapse collapse" aria-labelledby="flush-heading{{docs.number}}" data-bs-parent="#accordionFlushExample">
+                  <div id="flush-collapse{{docs.number}}" class="accordion-collapse collapse" style="padding: 15px;" aria-labelledby="flush-heading{{docs.number}}" data-bs-parent="#accordionFlushExample">
```

Comment:
avoid using inline styling as it gets a little difficult to track various different CSS changes. Instead, do all the styling changes in the designated folder.